src/plugins/intel_gpu/include/intel_gpu/op/gather_compressed.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "openvino/core/node.hpp"
+#include "openvino/op/gather.hpp"
+
+namespace ov {
+namespace intel_gpu {
+namespace op {
+
+class GatherCompressed : public ov::op::v8::Gather {
+public:
+    OPENVINO_OP("GatherCompressed", "gpu_opset");
+
+    GatherCompressed() = default;
+
+    GatherCompressed(const ov::Output<Node> &A,
+                     const ov::Output<Node> &B,
+                     const ov::Output<Node> &C,
```

Comment:
These args can have more meaningful names (data, indices, axis)

Reply:
I updated A, B, C to data, indices, axis respectively.